geomstats/learning/mdm.py

```diff
@@ -0,0 +1,86 @@
+"""The MDM classifier on manifolds."""
+
+from geomstats.learning.frechet_mean import FrechetMean
+import numpy
```

Comment:
Use:
```
import geomstats.backend as gs
```
instead of
```
import numpy
```
and `gs.` instead of `np.`. This will allow your code to work in numpy, tensorflow and pytorch seamlessly.

Reply:
Replaced `np` with `gs`.